docs/reference/modules/remote-clusters-shared.asciidoc

```diff
@@ -13,12 +13,12 @@ h| Remote cluster | 5.0->5.5 | 5.6 | 6.0->6.6 | 6.7 | 6.8 | 7.0 | 7.1->7.x
 |====
 
 ifeval::["{release-state}"!="released"]
-NOTE: This documentation is for {es} {version}, which is not yet released. The
-above compatibility table only applies if the local or remote cluster is running
-a released version of {es}. A local cluster running a pre-release build of {es}
-can also communicate with remote clusters running the same pre-release build.
-Running a mix of pre-release builds is unsupported and typically will not work,
-even if the builds are the same version.
+NOTE: This documentation is for {es} version {version}, which is not yet
+released. The above compatibility table applies if at least one of the clusters
```

Comment:
I wonder if this is strictly true. Say you have a 7.14-SNAPSHOT build and try to communicate to a 7.15 node. That is not guaranteed to work I think? So if one of them is a snapshot build, the other must be of lower version?

Reply:
Ah, good point (almost). It's not the version that matters, however, since an unreleased 7.last.x-SNAPSHOT build can legitimately communicate with all released 8.y.z builds. What matters is the build date I think, the -SNAPSHOT build must have happened after the release. I pushed 1d333a4a194.

(this solves the 7.14-SNAPSHOT vs 7.15 case too because we typically wouldn't be making a 7.14-SNAPSHOT build once 7.15 is released and even if we did they should remain wire-compatible).